tasks/src/depth-estimation/data.ts

```diff
@@ -29,7 +29,16 @@ const taskData: TaskDataCustom = {
 			id:          "vinvino02/glpn-kitti",
 		},
 	],
-	spaces:       [],
+	spaces:       [
+		{
+			description: "An application that predicts the depth of an image and then reconstruct the 3D model as voxels.",
+			id:          "radames/dpt-depth-estimation-3d-voxels",
```

Comment:
Maybe https://huggingface.co/spaces/radames/dpt-depth-estimation-3d-obj is a bit easier to understand (same concept, but using a point cloud)